src/tenant/tenant.resolver.ts

```diff
@@ -45,4 +45,35 @@ export default class TenantResolver {
 
     return tenant;
   }
+
+  @Mutation(() => Boolean)
+  async markRentAsLate(@Arg('id', () => Int) id: number): Promise<boolean> {
+    const t = await Tenant.findOne({where: {id}});
+    if (t) {
+      const l = await Lease.find({where: {tenantId: id}});
```

Comment:
here, we don't need to fetch all the leases concerning a user : in fact, we only need the number of **started** leases.  (we consider a user with finished lease can still be late with payment or marked as not late anymore)
I would suggest something like 
```suggestion
            const leaseCount = await Lease.count({
        where: {
          tenantId: t.id,
          start: LessThan(new Date()),
        },
      });
```

And then check leaseCount in the following `if`